src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
I don't think we should move `gin` to InitialSessionState.cs. I think for all aliases that are declared using `Alias()` attribute, we should keep them that way. They are not core cmdlets, and I don't see a problem to have them with 'None' option.